jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -92,6 +92,7 @@
 
   private static final String JIB_CLASSPATH_FILE = "jib-classpath-file";
   private static final String JIB_MAIN_CLASS_FILE = "jib-main-class-file";
+  private static final String DEFAULT_JIB_DIR = "src/main/jib";
```

Comment:
Not that this isn't going to work on Windows, but we just prefer this form:

```suggestion
  private static final Path DEFAULT_JIB_DIR = Paths.get("src").resolve("main").resolve("jib");
```